src/components/Book/SearchLandscapeBook.tsx

```diff
@@ -0,0 +1,399 @@
+        {' '}
+        {discount > 0 && (
+          <>
+            <Discount>
+              (
+              {Math.ceil(discount)}
+              %↓)
+            </Discount>
+              {' '}
+            <OriginalPrice>
+              {price.toLocaleString('ko-KR')}
+              원
+            </OriginalPrice>
+          </>
+        )}
+      </dd>
+    </PriceItem>
+  );
+}
+
+function PriceInfo(props: {
+  seriesPriceInfo: SearchTypes.SeriesPriceInfo[];
+  book: BookApi.ClientBook | null;
+  isTrial: boolean;
+}) {
+  if (!props.book) {
+    return null;
+  }
+  const { book, isTrial } = props;
+  const seriesPriceInfo: Record<string, SearchTypes.SeriesPriceInfo> = {};
+  props.seriesPriceInfo.forEach((info) => {
+    seriesPriceInfo[info.type] = info;
+  });
+  if (seriesPriceInfo.rent || seriesPriceInfo.normal) {
+    return (
+      <>
+        {seriesPriceInfo.rent && (
+          <PriceLabel
+            title="대여"
+            price={
+              seriesPriceInfo.rent.min_price !== 0
+                ? seriesPriceInfo.rent.min_price
+                : seriesPriceInfo.rent.max_price
+            }
+          />
+        )}
+        {seriesPriceInfo.normal && (
+          <PriceLabel
+            title="구매"
+            price={isTrial ? 0 : seriesPriceInfo.normal.min_nonzero_price}
+          />
+        )}
+      </>
+    );
+  }
+  if (book?.price_info) {
+    const { buy = null, rent = null } = book.price_info;
+    return (
+      <>
+        {rent && (
+          <PriceLabel
+            title="대여"
+            price={rent.regular_price}
+            discount={rent.discount_percentage}
+          />
+        )}
+        {buy && (
+          <PriceLabel
+            title="구매"
+            price={isTrial ? 0 : buy.regular_price}
+            discount={buy.discount_percentage}
+          />
+        )}
+      </>
+    );
+  }
+  return null;
+}
+
+interface SearchLandscapeBookProps {
+  item: SearchTypes.SearchBookDetail;
+  title: string;
+}
+
+function getLastVolumeId(item: SearchTypes.SearchBookDetail) {
+  if (item.is_series_complete || item.book_count === 1) {
+    return item.b_id;
+  }
+  if (item.opened_last_volume_id.length > 0) {
+    return item.opened_last_volume_id;
+  }
+  return item.b_id;
+}
+
+export function SearchLandscapeBook(props: SearchLandscapeBookProps) {
+  const { item, title } = props;
+  const thumbnailId = getLastVolumeId(item);
+  const book = useSelector((state: RootState) => state.books.items[item.b_id]);
+  if (book?.is_deleted) {
+    return null;
+  }
+  const {
+    parent_category,
+    parent_category2,
+    parent_category_name,
+    parent_category_name2,
+  } = item;
+  // Fixme desc.intro === '책 정보가 없습니다' 일 경우 처리 확인
+  const clearDesc = book?.clientBookFields?.desc?.intro
+      ?.replace(/[\r\n]/g, ' ')
+      ?.replace(/&#10;/g, ' ')
+      ?.replace(/&lt;/g, '<')
+      ?.replace(/&gt;/g, '>')
+      ?.replace(/(<([^>]+)>)/gi, '') ?? '';
```

Comment:
```suggestion
  const clearDesc = (book?.clientBookFields?.desc?.intro ?? '')
      .replace(/[\r\n]/g, ' ')
      .replace(/&#10;/g, ' ')
      .replace(/&lt;/g, '<')
      .replace(/&gt;/g, '>')
      .replace(/(<([^>]+)>)/gi, '');
```

이게 훨씬 나을것같네요 매번 분기 안쳐도되고